colcon.pkg

```diff
@@ -0,0 +1,8 @@
+# Configuration file for colcon (https://colcon.readthedocs.io).
+#
+# Please see the doc for the details of the spec:
+#   - https://colcon.readthedocs.io/en/released/user/configuration.html#colcon-pkg-files
+
+{
+  "dependencies": ["gz-cmake4"],
```

Comment:
```suggestion
  "build-dependencies": ["gz-cmake4"],
```
Quoting: https://colcon.readthedocs.io/en/released/user/configuration.html#colcon-pkg-files
> dependencies (list of strings) to declare additional dependencies on other packages. For more fine grain control it is also possible to set build-dependencies, run-dependencies, and test-dependencies.

Reply:
Tricky.

Is `gz-cmake4` required when a package which depends on `gz-tools` is built? If so, it would also need to be in `run-dependencies`.

Reply:
I'll leave this as is for now and merge to fix CI. We can follow up with other approaches after CI is fixed